src/docs/predicted_molecular_consequence.md

```diff
@@ -59,6 +61,8 @@ A set of information enabling interpretation of likely variant impact on genomic
 },
 "prediction_results": [] ,
 "transcript_biotype": "protein_coding",
+"exon_number": "2/5",
```

Comment:
The name of this field does not match the name in the table.

I also wonder about the purpose of these two fields. Is the web client expected to do anything with them? If so, is it expected to do anything with individual parts of this string? Even if this is used for display purposes alone, I wonder if Andrea might want us to format different parts of this string differently.